pkg/apis/v1alpha1/types.go

```diff
@@ -77,6 +78,7 @@ type PlacementRequestSpec struct {
 	// SchedulerName is the name of the scheduler that is responsible for
 	// creating this placement request. This is used to identify the
 	// queue that the placement request belongs to.
+	// +kubebuilder:validation:Required
```

Comment:
I am not sure whether this also means the values needs to be non-empty [1]:
```
// +kubebuilder:validation:Required
specifies that this field is required.
```
I think an empty value is still a valid value. `+kubebuilder:validation:items:MinLength=1` will make it guaranteed.

[1] https://book.kubebuilder.io/reference/markers/crd-validation

Reply:
You are absolutely right. There is a difference between not setting the property at all and setting it to an empty value. TIL. This should be fixed now.